src/buildDependencies.ts

```diff
@@ -0,0 +1,44 @@
+import core from '@actions/core'
+import { HttpClient } from '@sparkdotfi/common-universal/http-client'
+import { LogFormatterPretty, Logger, LoggerTransport } from '@sparkdotfi/common-universal/logger'
+import { Config, getConfig } from './config'
+import { ActionEnv } from './config/environment/ActionEnv'
+import { ReportSender } from './periphery/reporter/ReportSender'
+
+export interface Dependencies {
+  httpClient: HttpClient
+  reportSender: ReportSender
+  config: Config
+}
+
+export function buildDependencies() {
```

Comment:
IMO this should be called `buildActionDependencies` as it's specific to github action runner